lib/src/main/kotlin/com/swmansion/starknet/data/types/StarknetChainId.kt

```diff
@@ -18,16 +18,33 @@ data class StarknetChainId(val value: Felt) {
         @field:JvmField
         val INTEGRATION_SEPOLIA = StarknetChainId(Felt.fromHex("0x534e5f494e544547524154494f4e5f5345504f4c4941")) // encodeShortString('SN_INTEGRATION_SEPOLIA'),
 
+        /**
+         * Create [StarknetChainId] from a network name represented as ASCII string.
+         *
+         * @param networkName string transformed to [StarknetChainId]
```

Comment:
we generally did not informa about return value in other docstrings, right? (provider springs to my mind)
should we start doing so now?

Reply:
We normally do, but sometimes we don't (in `Felt` for example).

I just assumed 
```
* @return [StarknetChainId]
```
is redundant in the context.

Reply:
I stand corrected, indeed we have this for signer as well. We might want to think about adding the information to provider (and other relevant places) as well. thanks for clarification, this can be resolved.